.github/workflows/ci.yaml

```diff
@@ -96,7 +100,10 @@ jobs:
         uses: taiki-e/install-action@v2
         with:
           tool: cargo-insta
-      - uses: Swatinem/rust-cache@v2
+      - name: Set up basic Rust cache
+        uses: Swatinem/rust-cache@v2
```

Comment:
Should we remove `rust-cache` for the test to avoid double caching (or the two conflicting)?

Reply:
The pull request we were referencing (that showed a speedup) had both setup